examples/quickstarts/helidon-standalone-quickstart-mp/pom.xml

```diff
@@ -179,6 +180,14 @@
                         <mainClass>${mainClass}</mainClass>
                     </configuration>
                 </plugin>
+                <plugin>
```

Comment:
How about just defining the property `<maven.site.skip>true</maven.site.skip>`

Reply:
Fixed